draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -117,16 +116,31 @@ wherever possible.
 
 ## Feature Summary
 
-* This specification defines
- - a format for the payload of an Issuer-signed JWT containing selectively disclosable claims that include object properties (name-value pairs), array elements, and nested data structures built from these,
- - a format for data associated with the JWT that enables selectively disclosing those claims,
- - facilities for binding the JWT to a key and associated data to prove possession thereof, and
- - a format, extending the JWS Compact Serialization, for the combined transport of the JWT and associated data that is suitable for both issuance and presentation.
-* An alternate format utilizing the JWS JSON Serialization is also specified.
-* This specification enables combining selectively disclosable claims with
-  clear-text claims that are always disclosed.
-* For selectively disclosable claims that are object properties, both the key and value are always blinded.
-
+This specification defines two primary data formats:
+
+* SD-JWT: A format for signed, selectively-disclosable JSON content, together
+  with a set of selectively disclosed claims
+  - A format for the payload of an Issuer-signed JWT containing selectively
+    disclosable claims that include object properties (name-value pairs), array
+    elements, and nested data structures built from these
+  - A format for data associated with the JWT that enables selectively
+    disclosing those claims
+  - A format extending the JWS Compact Serialization, allowing for the combined
+    transport of the Issuer-signed JWT and the disclosure data for selectively
+    disclosed claims
+  - An alternate format extending the JWS JSON Serialization, also allowing for
+    transport of the Issuer-signed JWT and disclosure data
+
+* SD-JWT-KB: A format for proving posssion of a private key bound to an SD-JWT
```

Comment:
```suggestion
* SD-JWT-KB: A composite structure that is presented to the verifier that consists of the following:
```

Reply:
I would generally prefer that the `SD-JWT` and `SD-JWT-KB` entries be grammatically parallel, but I don't feel strongly.